modules/plugin-search.php

```diff
@@ -506,5 +508,28 @@ class="jetpack-plugin-search__dismiss"
 		return $links;
 	}
 
+}
 
+/**
+ * Master control that checks if Plugin search hints is active.
+ *
+ * @since 7.1.1
+ *
+ * @return bool True if PSH is active.
+ */
+function jetpack_is_psh_active() {
+	if ( false === ( $status = get_transient( 'jetpack_psh_status' ) ) ) {
+		$response = wp_remote_get( 'https://jetpack.com/psh-status/', array( 'sslverify' => false ) );
+		if ( is_wp_error( $response ) ) {
+			return false;
```

Comment:
Do we want to default to false?

Reply:
I don't know. I guess if this failed, it was because the A8c servers weren't working, which could only happen if the world was coming to an end. I'll switch to `true` by default.